pegasus/sites.v3/hourofcode.com/views/hoc_action_buttons.haml

```diff
@@ -1,7 +1,11 @@
 -if ["post-hoc", false].include?(hoc_mode)
   %a#tryitbutton.ctabuttonatag{href: resolve_url('/learn')}
     %button.ctabutton-white
-      =hoc_s(:front_start_learning)
+      =hoc_s(:front_join_us_try)
```

Comment:
```suggestion
      = hoc_s(:front_join_us_try)
```

Reply:
I will throw caution to the wind and do it though.